07a_system_activity_plots.R

```diff
@@ -149,6 +170,7 @@ system_activity_prep_summary <- reactive({
   inflow %>%
     full_join(outflow, join_by(Status, values, Time)) %>%
     full_join(frame_summary, join_by(Status, Time)) %>%
+    slice(1:3, 6, 5, 4) %>%
```

Comment:
The purpose of this is to sort the bars, right? Ideally, we'd order the bars using factors or sorting by name, so it's clear what's happening. Second best is e a comment explaining the purpose